### ISSUE TITLE
Topic 5 Knowledge Check test "typo" and possible answer error

### ISSUE THREAD

**Issue by Ailikekaus** [Section508Coordinators/DHSA-TT-200-A#164]
## Issue Type
- [ ] System Bug (the course doesn't function properly)
- [x] Typo/grammar
- [x] Incorrect content
- [ ] 508 issue (something is not accessible according to our own test process)
- [ ] Other, specify...[replace this text with your issue type description]

## Description
1. Test Name Typo
2. Answer options all false

## Location
https://training.section508testing.net/mod/quiz/attempt.php?attempt=27876&page=8
Topic 5 Forms - Knowledge Check Quiz
Question 8

## Actual Behavior / Observation
1. Test Name is listed as `3.3.4-input-instructions` for ALL questions, but I'm pretty sure it should be `3.3.4-error-prevention`.
![image](https://user-images.githubusercontent.com/19952958/59865723-c8188a00-9357-11e9-9269-226139d168fc.png)

2. The course lists "Submittals of user test responses." as the correct answer for the below question, but the test script and course content state user test responses are DNA for 3.3.4-error-prevention.
![image](https://user-images.githubusercontent.com/19952958/59865765-eaaaa300-9357-11e9-8ec8-b6df238a1652.png)

```
What content needs to be tested for 3.3.4-input-instructions? [s/b 3.3.4-error-prevention]
Select one:
a. Submittals of online parent satisfaction survey responses.
b. Submittals of a non-binding memorandum of understanding.
c. Submittals of a non-binding contract.
d. Submittals of user test responses.  
```
![image](https://user-images.githubusercontent.com/19952958/59865742-d8306980-9357-11e9-80cd-c48ababc68fa.png)

## Expected Behavior / Recommendation
[Describe what you expected to happen and/or your recommendation to fix or correct the issue]

## Steps to Reproduce the Issue (if applicable)
N/A

**Comment by AnnMarieDavis** [comment]
This test name was properly changed in the question bank for Learning Objective #1, but needed to be done for Learning Objective #2, as pointed out by this pilot participant. The test name was updated in the KC questions to 3.3.4-error-prevention.

Clarification is still needed for when submission of test question should be DNA and when it applies in the course and in the KC questions.

**Comment by drewnielson** [comment]
Unfortunately, we have created confusion due to the double negative: the test **DOES NOT APPLY** if the page does **NOT** submit test responses. So if it **DOES** submit test responses, then the test applies. Closing for now since the first issue is fixed. However, will keep on the radar to improve the wording to avoid the double negative. Will also look for other places in the test process where we likely have the same issue.
